.github/workflows/format.yml

```
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  format:
+    name: formatter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+      - run: npm install prettier
+      - run: node ./node_modules/.bin/prettier '**/*.md'
```

上書きするにはwriteオプションが必要みたいです！

```suggestion
      - run: node ./node_modules/.bin/prettier --write '**/*.md'
```

---

ありがとうございます！
直しておきました～